convert/convertBooks.ts

```diff
@@ -201,8 +201,9 @@ export async function convertBooks(
                             'quizzes',
                             book.id + '.json'
                         ),
-                        content: JSON.stringify(convertQuizBook(context, book))
+                        content: JSON.stringify(convertQuizBook(context, book), null, 2)
                     });
+                    process.stdout.write(` ${book.id}`);
```

Comment:
Can't put a comment on line 255, but it crashes there if the project only contains quizzes (At least that's what I think did it.  A project with two quizzes in the main book collection and that is all).  Entries contains a single entry with no docSets, so docSets[0] didn't exist so it blew up referencing its id.  A project with only quizzes is a valid combination and can't crash on startup.

Reply:
There is a separate PR for this: #580.  It needs closer review.